src/main/java/com/health/controller/AjaxController.java

```diff
@@ -1189,6 +1189,98 @@ private HashMap<String, String> setResponse(Integer status) {
 
     }
 
+    @RequestMapping("/loadLanguageAndTopicByCategoryResource")
+    public @ResponseBody ArrayList<HashMap<Integer, String>> getLanguageByCategoryResourse(
+            @RequestParam(value = "catId") int catId) {
+
+        ArrayList<HashMap<Integer, String>> arlist = new ArrayList<>();
+        HashMap<Integer, String> languagesMap = new HashMap<>();
+        HashMap<Integer, String> topicMap = new HashMap<>();
+        HashMap<Integer, String> tutorialCount = new HashMap<>();
+
+        Category cat = catService.findByid(catId);
+        logger.info("Variables of loadLanguageByCategoryResourse cat : {}", cat);
+        if (cat != null) {
+            List<TopicCategoryMapping> tcm = topicCatService.findAllByCategory(cat);
+            List<ContributorAssignedTutorial> conList = conService.findAllByTopicCat(tcm);
+            List<Tutorial> tutList = tutService.findAllByconAssignedTutorialAndStatus(conList);
+
+            for (Tutorial tut : tutList) {
+                Category cat1 = tut.getConAssignedTutorial().getTopicCatId().getCat();
+                if (cat1.isStatus())
```

Comment:
This check is not required as we are starting with an enabled category. Remove it.